server/src/main/java/org/opensearch/http/HttpTracer.java

```diff
@@ -116,10 +116,11 @@ void traceResponse(
     ) {
         logger.trace(
             new ParameterizedMessage(
-                "[{}][{}][{}][{}][{}] sent response to [{}] success [{}]",
+                "[{}][{}][{}][{}][{}][{}] sent response to [{}] success [{}]",
                 requestId,
                 opaqueHeader,
                 restResponse.status(),
+                restResponse.status().getStatus(),
```

Comment:
This seems redundant? I think every RestStatus can be mapped to a unique integer code. Why do we need to also write that integer into this log?

Reply:
Yeah on high level it looks redundant, but there can be use cases where if user wants to configure and trigger an alarm if they see very high 5xx requests in logs. Without having the Response's integer code in log it would be very hard to add filter for RestStatus from 500 -599 as each will have different names. Using filter on Integer codes will be easier as to filter all 500 -599 responses we can just use 5xx.